src/mapml.css

```
@@ -776,3 +776,45 @@ label.mapml-layer-item-toggle {
   ) {
   outline: 0!important;
 }
+
+
+/**
+* Feature Index
+ */
+.mapml-feature-index-box:not([hidden]) {
+  margin: -36px 0 0 -36px;
+  width: 72px;
+  height: 72px;
+  left: 50%;
+  top: 50%;
+  content: '';
+  display: block;
+  position: absolute;
+  z-index: 10000;
+}
+
+.mapml-feature-index {
```

```suggestion
.mapml-feature-index:not([hidden]) {
```

Oops I missed this one. Same as https://github.com/Maps4HTML/Web-Map-Custom-Element/pull/636#pullrequestreview-834977702.